models/helper/MarginHelper.py

```diff
@@ -0,0 +1,61 @@
+def calculate_margin(buy_size: float = 0.0, buy_filled: int = 0.0, buy_price: int = 0.0, buy_fee: float = 0.0,
```

Comment:
Would it be better if we put overridden calculate_margin function under the API's, so it would called from app(PyCryptoBot) like getTakerFee() funtions.

And this way we can reduce parameter counts since fee functions will be called from APIs it selves.

Reply:
I agree, we can do that once everything's working as expected.